<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>851</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="summaryPage">
     <attribute name="title">
      <string>Yhteenveto</string>
     </attribute>
     <widget class="QTableWidget" name="meatSharedTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="groupSummaryTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>331</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sharedMeatLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jaetut lihat ryhmittäin</string>
      </property>
     </widget>
     <widget class="QLabel" name="groupSummaryLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ryhmän tiedot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="summarRefreshPushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Päivitä</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="summarySuggestGroupBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>461</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Ehdotukset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="killPage">
     <attribute name="title">
      <string>Kaato</string>
     </attribute>
     <widget class="QComboBox" name="shotByComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Karttaan merkitty paikannimi</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaataja</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="shotDateEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Finnish" country="Finland"/>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>11</month>
        <day>15</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaatopäivä</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ageGroupComboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Eläin</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ikäryhmä</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sukupuoli</string>
      </property>
     </widget>
     <widget class="QComboBox" name="genderComboBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="usageComboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="weightLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Paino</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Käyttö</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="additionalInfoTextEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Tähän vieraan kaatajan nimi tai havaintoja eläimestä</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lisätietoja</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="killsKillsWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>541</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaadot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveShotPushButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tallenna</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="locationLineEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Paikka</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sharePage">
     <attribute name="title">
      <string>Lihanjako</string>
     </attribute>
     <widget class="QTableWidget" name="sharedKillsTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>681</width>
        <height>192</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Valitse kaato, jonka lihoja jaetaan</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotByLabel_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaadot</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="shareDateEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Finnish" country="Finland"/>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>11</month>
        <day>15</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jakopäivä</string>
      </property>
     </widget>
     <widget class="QComboBox" name="portionComboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="amountLineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ruhon osa</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Määrä (KG)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="groupComboBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotDateLabel_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jakoryhmä</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shareSavePushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tallenna</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="shareSuggestGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>681</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Jakoehdotukset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="licensePage">
     <attribute name="title">
      <string>Luvat</string>
     </attribute>
     <widget class="QLineEdit" name="licenseYearLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Lupavuosi</string>
      </property>
     </widget>
     <widget class="QComboBox" name="licenseAnimalComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Eläin</string>
      </property>
     </widget>
     <widget class="QComboBox" name="licenseAgeGroupComboBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Ikäryhmä</string>
      </property>
     </widget>
     <widget class="QComboBox" name="licenseGenderComboBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Sukupuoli</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="licenseAmountLineEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Määrä</string>
      </property>
     </widget>
     <widget class="QPushButton" name="licenseSavePushButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tallenna</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="licenseSummaryTableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>671</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Myönnetyt luvat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="maintenancePage">
     <attribute name="title">
      <string>Ylläpito</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Seura</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Seurue</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Jakoryhmä</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Jäsen</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>summarRefreshPushButton</tabstop>
  <tabstop>meatSharedTableWidget</tabstop>
  <tabstop>groupSummaryTableWidget</tabstop>
  <tabstop>shotByComboBox</tabstop>
  <tabstop>shotDateEdit</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>animalComboBox</tabstop>
  <tabstop>ageGroupComboBox</tabstop>
  <tabstop>genderComboBox</tabstop>
  <tabstop>weightLineEdit</tabstop>
  <tabstop>usageComboBox</tabstop>
  <tabstop>additionalInfoTextEdit</tabstop>
  <tabstop>saveShotPushButton</tabstop>
  <tabstop>killsKillsWidget</tabstop>
  <tabstop>sharedKillsTableWidget</tabstop>
  <tabstop>shareDateEdit</tabstop>
  <tabstop>portionComboBox</tabstop>
  <tabstop>amountLineEdit</tabstop>
  <tabstop>groupComboBox</tabstop>
  <tabstop>shareSavePushButton</tabstop>
  <tabstop>licenseYearLineEdit</tabstop>
  <tabstop>licenseAnimalComboBox</tabstop>
  <tabstop>licenseAgeGroupComboBox</tabstop>
  <tabstop>licenseGenderComboBox</tabstop>
  <tabstop>licenseAmountLineEdit</tabstop>
  <tabstop>licenseSavePushButton</tabstop>
  <tabstop>licenseSummaryTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
